cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -453,6 +453,118 @@ describe("Timeline", () => {
             cy.get(".mx_RoomView_searchResultsPanel").percySnapshotElement("Highlighted search results");
         });
 
+        it("should find words in various languages and highlight them on search result", () => {
+            // "Test" in various languages
+            const stringAr = "اِمْتِحَان‎";
+            const stringEl = "εξέταση";
+            const stringHe = "מִבְחָן";
+            const stringHi = "आज़माइश";
+            const stringJa = "テスト";
+            const stringKo = "테스트";
+            const stringTh = "การสอบ";
+            const stringZh = "测试";
+
+            const MESSAGE = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor";
+
+            const sendString = (string: string) => {
+                cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                    msgtype: "m.text" as MsgType,
+                    body: string,
+                });
+            };
+
+            const searchAndHighlight = (string: string) => {
+                cy.get(".mx_SearchBar_input input").clear().invoke("val", string).trigger("input").type("{enter}");
+                cy.get(".mx_EventTile:not(.mx_EventTile_contextual) .mx_EventTile_searchHighlight").should("exist");
+            };
+
+            // Open roomId room
+            cy.visit("/#/room/" + roomId);
+
+            // Wait until configuration is finished
+            cy.contains(
+                ".mx_RoomView_body .mx_GenericEventListSummary .mx_GenericEventListSummary_summary",
+                "created and configured the room.",
+            ).should("exist");
+
+            // Create a bot "BotBob" and invite it
+            let bot: MatrixClient;
+            cy.getBot(homeserver, {
+                displayName: "BotBob",
+                autoAcceptInvites: false,
+            }).then((_bot) => {
+                bot = _bot;
+                cy.inviteUser(roomId, bot.getUserId());
+                bot.joinRoom(roomId);
+
+                // Make sure the bot joined the room
+                cy.contains(
+                    ".mx_GenericEventListSummary .mx_EventTile_info.mx_EventTile_last",
+                    "BotBob joined the room",
+                ).should("exist");
+
+                // Have bot send MESSAGE to roomId
+                cy.botSendMessage(bot, roomId, MESSAGE);
+
+                // Have bot send stringJa to roomId
+                cy.botSendMessage(bot, roomId, stringJa);
```

Comment:
why do we send `stringJa` twice?

Reply:
intended to ensure that stringJa from both users can be searched and found. ok to remove

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/10319/commits/e64b3de23f46025e26fd8ea213767569b10c1ec2